docs/docs.scss

```diff
@@ -1,22 +1,104 @@
 @import "../styles/style-guide.scss";
 @import "../bower_components/pygments/css/default.css";
 
-.swatch {
+$swatch-padding: 5px;
+$swatch-width-height: 90px;
+$swatch-label-height: 30px;
+
+.swatch-with-label {
     display:inline-block;
-    width: 6rem;
-    height: 6rem;
-    border-radius: 0.25rem;
-    border-color: black;
+    width: $swatch-width-height;
+    height: $swatch-width-height + $swatch-label-height + 2px;
+    vertical-align: top;
+    margin: 4px;
+
+    label{
+        background-color:white;
+        width:$swatch-width-height;
```

Comment:
Properties should be ordered background-color, display, font-size, font-weight, height, margin, text-align, width<br>Colon after property should be followed by one space